packages/integration-tests/tests/api/transactions.test.ts

```diff
@@ -109,736 +109,1071 @@ describe("Transactions", () => {
-        expect(response.body.items[3]).toStrictEqual(expect.objectContaining({ type: "refund" }));
+        expect(response.body.items[3]).toStrictEqual(
+          expect.objectContaining({ tokenAddress: Token.ETHER_ERC20_Address })
+        );
+        expect(response.body.items[3]).toStrictEqual(expect.objectContaining({ type: TransactionsType.refund }));
+        expect(response.body.items[3]).toStrictEqual(
+          expect.objectContaining({
+            token: {
+              decimals: 18,
+              iconURL: "https://assets.coingecko.com/coins/images/279/large/ethereum.png?1698873266",
+              l1Address: Token.ETHER_Address,
+              l2Address: Token.ETHER_ERC20_Address,
+              liquidity: 220000000000,
+              symbol: "ETH",
+              name: "Ether",
+              usdPrice: 1800,
+            },
+          })
+        );
+        expect(response.body.items[3]).toStrictEqual(expect.objectContaining({ tokenType: "ETH" }));
+        expect(response.body.meta).toStrictEqual(expect.objectContaining({ totalItems: 4 }));
+        expect(response.body.meta).toStrictEqual(expect.objectContaining({ itemCount: 4 }));
+        expect(response.body.meta).toStrictEqual(expect.objectContaining({ itemsPerPage: 10 }));
+        expect(response.body.meta).toStrictEqual(expect.objectContaining({ totalPages: 1 }));
+        expect(response.body.meta).toStrictEqual(expect.objectContaining({ currentPage: 1 }));
+        expect(response.body.links).toStrictEqual(
+          expect.objectContaining({ first: `${decapitalizedAddress}?limit=10` })
+        );
+        expect(response.body.links).toStrictEqual(expect.objectContaining({ previous: "" }));
+        expect(response.body.links).toStrictEqual(expect.objectContaining({ next: "" }));
+        expect(response.body.links).toStrictEqual(
+          expect.objectContaining({ last: `${decapitalizedAddress}?page=1&limit=10` })
+        );
       });
     });
-  });
 
-  describe("/transactions/{transactionHash}", () => {
-    beforeAll(async () => {
-      const customToken = await helper.getStringFromFile(bufferFile + Buffer.L2deposited);
-      await playbook.transferFailedState(customToken);
-    });
-
-    //@id1460
-    it("Verify the ETH withdrawal to the other address via /transactions/{transactionHash}", async () => {
-      await helper.retryTestAction(async () => {
-        txHash = await helper.getStringFromFile(bufferFile + Buffer.txEthWithdrawOtherAddress);
-        apiRoute = `/transactions/${txHash}`;
-        response = await helper.performGETrequest(apiRoute);
-
-        expect(response.status).toBe(200);
-        expect(response.body.hash).toBe(txHash);
-        expect(response.body.to).toBe("0x000000000000000000000000000000000000800A");
-        expect(response.body.from).toBe(Wallets.richWalletAddress);
-        expect(response.body.value).toBe("9000000000000");
+    describe("/transactions/{transactionHash}", () => {
```

Comment:
Do we really need this tab? Do we have describe("/transactions/{transactionHash}" inside of describe("/transactions/{transactionHash}/transfers" ?

Reply:
Fixed